R/theme-defaults.r

```diff
@@ -117,7 +117,7 @@ theme_grey <- function(base_size = 11, base_family = "") {
     panel.ontop    =     FALSE,
 
     strip.background =   element_rect(fill = "grey85", colour = NA),
-    strip.text =         element_text(colour = "grey10", size = rel(0.8)),
+    strip.text =         element_text(colour = "grey10"),
```

Comment:
So we will remove the `rel` size in animint2 aesthetic?

Reply:
Thanks for your review @Faye-yufan and for the fix in the tests.
I'm not sure I understand the question. I think rel is ok to use still, but this deletion of strip.text property was intended to increase the text size of the strip text, does that make sense?

Reply:
Yeah sure! I get it now.